.circleci/config.yml

```
@@ -195,6 +198,9 @@ workflows:
       - test-lint-changelog:
           requires:
             - prep-deps
+      - test-e2e-chrome-webpack:
+          requires:
+            - prep-build-test-webpack
```

I'm wondering if we want to add the e2e quality gate for the e2e chrome in webpack, as we do with the other builds. I think we would want to, in that case, we could do this:

```suggestion
            - prep-build-test-webpack
            - get-changed-files-with-git-diff
```